src/usr/local/www/system_advanced_notifications.php

```diff
@@ -230,93 +230,92 @@
 
 $section->addInput(new Form_Input(
 	'smtpipaddress',
-	'E-Mail server',
+	'SMTP server',
 	'text',
 	$pconfig['smtpipaddress']
-))->setHelp('This is the FQDN or IP address of the SMTP E-Mail server to '.
+))->setHelp('The FQDN or IP address of the SMTP mail server to '.
 	'which notifications will be sent.');
 
 $section->addInput(new Form_Input(
 	'smtpport',
-	'SMTP Port of E-Mail server',
+	'SMTP port',
 	'number',
 	$pconfig['smtpport']
-))->setHelp('This is the port of the SMTP E-Mail server, typically 25, 587 '.
+))->setHelp('Port that the SMTP server accepts connections on. Typically 25, 587 '.
 	'(submission) or 465 (smtps).');
 
-$section->addInput(new Form_Input(
-	'smtptimeout',
-	'Connection timeout to E-Mail server',
-	'number',
-	$pconfig['smtptimeout']
-))->setHelp('This is how many seconds it will wait for the SMTP server to connect. Default is 20s.');
-
-$group = new Form_Group('Secure SMTP Connection');
-$group->add(new Form_Checkbox(
+$section->addInput(new Form_Checkbox(
 	'smtpssl',
-	'Enable SSL/TLS',
+	'SSL/TLS encryption',
 	'Enable SMTP over SSL/TLS',
-	isset($pconfig['smtpssl'])
+	$pconfig['smtpssl']
 ));
```

Comment:
We should probably add a note here that STARTTLS will only happen when this is **disabled**, and that it's different than this. Maybe:
> Enables direct SSL/TLS communication with the SMTP server, typically on port 465. When **disabled**, notifications will automatically attempt to use STARTTLS instead if supported by the SMTP server.